lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -382,6 +399,7 @@ class CategoryRenderer {
     /** @type {Map<TopLevelClumpId, Array<LH.ReportResult.AuditRef>>} */
     const clumps = new Map();
     clumps.set('failed', []);
+    clumps.set('warning', []);
```

Comment:
should add 'warning' to the list of statuses in the method jsdoc for `render()` above